gitclone/gitclone.go

```diff
@@ -293,16 +198,19 @@ func Execute(cfg Config) error {
 		tracker.logSubmoduleUpdate(updateTime)
 	}
 
-	if ref := checkoutStrategy.getBuildTriggerRef(); ref != "" {
-		fmt.Println()
-		logger.Infof("Exporting commit details")
-		if err := exportCommitInfo(gitCmd, ref, isPR, maxEnvLength); err != nil {
-			return newStepError("export_envs_failed", err, "Exporting envs failed")
-		}
-	} else {
-		fmt.Println()
-		logger.Warnf(`Can not export commit information like commit message and author as it is not available.
-This may happen when using Bitbucket with the "Manual merge" input set to 'yes' (using a Diff file).`)
```

Comment:
This looks like a suggestion, but it's not, because even if manual_merge was set to "yes", the step decided to override it and fetch the PR diff from Bitbucket directly. So there is no real solution to this problem with step inputs, the only workaround AFAIK is to rely on the env vars that are based on the webhook data.